src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +121,28 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Simple check that validates label is not empty string after trimming.
+     */
+    public static String parseLabel(String value) throws ParseException {
+        requireNonNull(value);
+        String trimmedValue = value.trim();
+        if (trimmedValue.equals("")) {
+            throw new ParseException("empty label");
```

Comment:
Should the exception message be declared in its own class (e.g. Label) so that the code is more standardised?